python/mxnet/optimizer.py

```diff
@@ -345,8 +345,8 @@ def __init__(self, learning_rate=0.002,
                  decay_factor=(1 - 1e-8),
                  wd=0.,
                  rescale_grad=1, clip_gradient=None,
-                 lr_scheduler=None):
-        super(Adam, self).__init__(rescale_grad)
+                 lr_scheduler=None, arg_names=None):
+        super(Adam, self).__init__(rescale_grad, arg_names=None)
```

Comment:
I am asking myself the same question! An accidental copy-paste probably. Fixed it!